CHANGELOG.md

```diff
@@ -7,16 +7,23 @@
 > * [CHANGELOG_1.1x.md](./documentation/changelog/CHANGELOG_1.1x.md) - v1.10.0 to v1.19.0
 > * [CHANGELOG_1.2x.md](./documentation/changelog/CHANGELOG_1.2x.md) - v1.20.0 to v1.29.2
 
-# UNRELEASED
+# Node v1.33.0-rc1 / 2025-05-01
```

Comment:
Do we aim to land F3 embedded manifests in this release too? if so I think it deserve s an explicit callout.

Reply:
Is there an issue or a merged PR for this? The release candidate was created from the master branch this morning. If we intend to add it later and backport it, I suggest we wait to highlight it until it has been merged and backported.

Reply:
I see we have it listed as a bonus for the RC1 here: https://github.com/filecoin-project/lotus/issues/13004, but it did not make it for the RC1-train, I think we can add it as a dependency for RC2 though if it is not too far away.